### ISSUE TITLE
Multiclass dice score

### ISSUE THREAD

**Issue by leosmaia21** [milesial/Pytorch-UNet#440]
Hello
I am pretrainig the model on 20K images from cocodataset, and right after the first epoch i have a dice score of 0.91, but i tested the model and it doest work well, the mask returned all black.

Thanks
![image](https://user-images.githubusercontent.com/77647786/235829471-8768f171-fa77-44e4-8be4-d386e7871501.png)

**Comment by ahmetfurkaann** [comment]
I have the same error. what is the problem with this
